Yona/Yona/Challenges/TimeFrameTimeZoneChallengeViewController.swift

```diff
@@ -242,6 +246,8 @@ extension TimeFrameTimeZoneChallengeViewController {
                 self.setChallengeButton.alpha = 0.5
             }
             self.tableView.endUpdates()
+            
+            NSTimer.scheduledTimerWithTimeInterval(0.1, target: self, selector: Selector.tableReload, userInfo: nil, repeats: false)
```

Comment:
I'm not sure its a good way, to handle this.
I think you should reconsider if it can be done without the timer.